llama_stack/cli/stack/run.py

```diff
@@ -59,6 +66,11 @@ def _add_arguments(self):
             help="Image Type used during the build. This can be either conda or container or venv.",
             choices=[e.value for e in ImageType],
         )
+        self.parser.add_argument(
+            "--no-ui",
+            action="store_true",
+            help="Don't start the UI server",
+        )
```

Comment:
We need a corresponding StackRunConfig update to control this via the config file.